ocp_resources/resource.py

```diff
@@ -107,15 +113,24 @@ def get_client(
             )
         )
     try:
-        # Ref: https://github.com/kubernetes-client/python/blob/v26.1.0/kubernetes/base/config/__init__.py
-        LOGGER.info("Trying to get client via new_client_from_config")
-
         # kubernetes.config.kube_config.load_kube_config sets KUBE_CONFIG_DEFAULT_LOCATION during module import.
         # If `KUBECONFIG` environment variable is set via code, the `KUBE_CONFIG_DEFAULT_LOCATION` will be None since
         # is populated during import which comes before setting the variable in code.
         config_file = config_file or os.environ.get("KUBECONFIG", "~/.kube/config")
+        client_configuration = client_configuration or client.Configuration()
+        proxy = os.environ.get("HTTPS_PROXY") or os.environ.get("HTTP_PROXY")
```

Comment:
User will give as configured `client_configuration` and we just use it.

Reply:
It will make the implementation very complicated on the caller side if we omit this important option.
Please consider keeping the proxy to be set additionally as environment variable.

Reply:
I can add logic to override the proxy part if it's already being configured in the  client_configuration regardless of being set by environment variable.